src/commands/apm_cmds/version.js

```diff
@@ -1,7 +1,31 @@
-const fs = require('fs')
+const { ensureWeb3 } = require('../../helpers/web3-fallback')
+const fs = require('fs-extra')
 const findUp = require('find-up')
 const semver = require('semver')
+const APM = require('@aragon/apm')
 
+const bumpVersionAndUpdateManifest = async ({ module, bump, network, apmOptions, cwd }) => {
+
+  const web3 = await ensureWeb3(network)
+  const apm = APM(web3, apmOptions)
+
+  let repo = { version: '0.0.0' }
+  try {
+    repo = await apm.getLatestVersion(module.appName)
+  } catch (e) {
+
+  }
+
+  const manifestLocation = await findUp('arapp.json', { cwd })
+  const manifest = fs.readJsonSync(manifestLocation)
+
+  manifest.version = semver.valid(bump) ? semver.valid(bump) : semver.inc(repo.version, bump)
+
+  fs.writeJsonSync(manifestLocation, manifest)
```

Comment:
We should still indent the JSON file as before

Reply:
Yes, good point! 👍